util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -227,90 +236,90 @@ command.playerNotOnline = {0} is not currently online
 
-command.error.invalidTimePeriod = Invalid time period '{0}'
-command.error.invalidNumber = Invalid number '{0}'
-command.error.invalidPage = There is no page {0}. Pages run from 1 to {1}.
-command.error.emptyResult = Empty result
+command.emptyResult = Empty result
 
 
 +           -----DESTROY THE CORE-----
 
-map.gamemode.short.dtc = DTC
-map.gamemode.long.dtc = Destroy the Core
-match.core.damageOwn = You may not damage your own core.
+core.name.short = DTC
+core.name.long = Destroy the Core
+
+core.damageOwn = You may not damage your own core.
 
 # {1} = core name (e.g "Left core")
 # {2} = team
-match.complete.core = {2}'s {1} has leaked
+core.leaked = {2}'s {1} has leaked
 
 # {1} = core name (e.g "Left core")
 # {2} = team
-match.touch.core.owner = {2}'s {1} has been damaged
+core.touch.owned.generic = {2}'s {1} has been damaged
 
 # {1} = core name (e.g "Left core")
 # {2} = team
-match.touch.core.owner.you = You damaged {2}'s {1}
+core.touch.owned.you = You damaged {2}'s {1}
 
 # {0} = player
 # {1} = core name (e.g "Left core")
 # {2} = team
-match.touch.core.owner.toucher = {2}'s {1} damaged by {0}
+core.touch.owned.player = {2}'s {1} damaged by {0}
 
 # {0} = player
 # {1} = core name (e.g "Left core")
-match.touch.core.toucher = {1} damaged by {0}
+core.touch.generic = {1} damaged by {0}
 
 +           -----DESTROY THE MONUMENT-----
 
-map.gamemode.long.dtm = Destroy the Monument
-map.gamemode.short.dtm = DTM
+destroyable.name.short = DTM
+destroyable.name.long = Destroy the Monument
 
 # {0} = player
 # {1} = monument name (e.g "Monument A")
 # {2} = team
-match.complete.destroyable = {2}'s {1} destroyed by {0}
+destroyable.complete.owned.player = {2}'s {1} destroyed by {0}
 
 # {1} = monument name (e.g "Monument A")
 # {2} = team
-match.touch.destroyable.owner = {2}'s {1} has been damaged
-
+destroyable.touch.owned.generic = {2}'s {1} has been damaged
 # {1} = monument name (e.g "Monument A")
 # {2} = team
-match.touch.destroyable.owner.you = You damaged {2}'s {1}
-
+destroyable.touch.owned.you = You damaged {2}'s {1}
 # {0} = player
 # {1} = monument name (e.g "Monument A")
 # {2} = team
-match.touch.destroyable.owner.toucher = {2}'s {1} damaged by {0}
-
+destroyable.touch.owned.player = {2}'s {1} damaged by {0}
 # {0} = player
 # {1} = monument name (e.g "Monument A")
-match.touch.destroyable.toucher = {1} damaged by {0}
+destroyable.touch.generic = {1} damaged by {0}
 
 +           -----DEATH-----
 
 death.respawn.unconfirmed = Left click to respawn
-
 # {0} = an amount of seconds
 death.respawn.unconfirmed.time = Left click to respawn in {0}s
-
 # {0} = an amount of seconds
 death.respawn.confirmed.time = Respawning in {0}s
 death.respawn.confirmed.waiting = You will respawn as soon as possible...
 
-# A prefix used if a player logs out before they die TODO: Rephrase
+death.respawn.confirmed.waiting.flagDropped = You will respawn when the flag is dropped...
```

Comment:
How about just `respawn` instead of `death.respawn`

Reply:
I would, but i cant find where `death.respawn.confirmed.waiting.flagDropped` is used, so i'm suspecting a builder somewhere.